pkg/services/ngalert/api/tooling/definitions/alertmanager.go

```diff
@@ -746,13 +746,43 @@ func (c *Config) UnmarshalJSON(b []byte) error {
 	if len(c.Route.Match) > 0 || len(c.Route.MatchRE) > 0 {
 		return fmt.Errorf("root route must not have any matchers")
 	}
+	if len(c.Route.MuteTimeIntervals) > 0 {
+		return fmt.Errorf("root route must not have any mute time intervals")
+	}
 
 	for _, r := range c.InhibitRules {
 		if err := r.UnmarshalYAML(noopUnmarshal); err != nil {
 			return err
 		}
 	}
 
+	tiNames := make(map[string]struct{})
+	for _, mt := range c.MuteTimeIntervals {
+		if mt.Name == "" {
+			return fmt.Errorf("missing name in mute time interval")
```

Comment:
Perhaps we should include the index in the error message so it's easier to find the invalid mute timing?

```suggestion
			return fmt.Errorf("missing name in mute time interval at index %d", ix)
```

Reply:
This is sensible, I'll change it 👍

Reply:
I just thought about this. As we want to move those changes upstream, the error message would then change back again. So I will stick to as it is.